php-fpm/recipes/configure.rb

```diff
@@ -69,6 +69,23 @@
   end
 end
 
+#
+# An attempt to make sure that the php prefix given is the prefix
+# chosen for php-cli at the end of an install
+php_package_prefix = node['php']['ppa']['package_prefix'].gsub(/php/, '')
```

Comment:
You could re-use `php_version` variable from:
https://github.com/till/easybib-cookbooks/blob/8736c1d436c0db0b68f6e5341a596755beb33205/php-fpm/recipes/configure.rb#L54

So just a few lines up.

Otherwise 👍